client/src/pages/Login.js

```diff
@@ -40,12 +45,30 @@ class LoginPage extends Component {
 
   handleSubmit = event => {
     event.preventDefault();
-    const isValid = this.validate();
-    if (isValid) {
-      console.log(this.state);
-      this.setState(initalState);
+    this.validate();
+
+    const { email, password } = this.state;
+
+    const data = {
+      email: email,
+      password: password
     }
+
+    axios.post('/users/login', data)
```

Comment:
You need to let axios know where exactly to send the info, so this should be: axios.post("http://localhost:3001/users/login", ...)

Reply:
We added "proxy": "http://localhost:3001" in package.json in client folder. So, we don't need to specify "http://localhost:3001 ??

Reply:
Oh, in that case this is good